app/Http/Requests/ProfileRequest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class ProfileRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'email' => 'required|max:255|min:10',
```

Comment:
đậy thêm ntn nữa cho cả hai bên, client lẫn server cho an toàn nha, vì nếu nó post bằng một công cụ khác thì server vẫn nhận nha: 'email' => 'required|email|unique:users',